src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
-                <ZUILogo beta={true} htmlColor="#ED1C55" size={40} />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href="/organize/" passHref>
-              <IconButton
-                aria-label="Home"
-                className={classes.roundButton}
-                color={key === '' ? 'primary' : 'secondary'}
-                data-test="home-button"
-                size="large"
-              >
-                <Home />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/people`} passHref>
-              <IconButton
-                aria-label="People"
-                className={classes.roundButton}
-                color={key.startsWith('/people') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <People />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/journeys`} passHref>
-              <IconButton
-                aria-label="Journeys"
-                className={classes.roundButton}
-                color={key.startsWith('/journeys') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <Explore />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/areas`} passHref>
-              <IconButton
-                aria-label="Areas"
-                className={classes.roundButton}
-                color={key.startsWith('/areas') ? 'primary' : 'secondary'}
-                data-test="area-button"
-                size="large"
-              >
-                <Map />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = () => {
+    setOpen(!open);
+    setExpanded(!expanded);
+  };
+
+  const model = useModel((env) => new OrganizationsDataModel(env));
+
+  const menuItemsMap = [
+    { icon: <Map />, name: 'areas' },
+    { icon: <Home />, name: 'home' },
+    { icon: <Explore />, name: 'journeys' },
+    { icon: <People />, name: 'people' },
+    { icon: <Event />, name: 'projects' },
+  ] as const;
+
+  const OrganizeSideBar = (
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
```

Comment:
What's the purpose of this (in this context)?

Reply:
The drawer code I got as an example for MUI library had it, but now I see I don't need it, thanks for pointing it out!